### PR TITLE
Node version/update node version/ma 3786

### PR DESCRIPTION
**Pull request by bhargavapratik**
Node version/update node version/ma 3786

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -108,7 +108,7 @@ Resources:
     Properties:
       FunctionName: SungardAS-Alerts
       Handler: index.handler
-      Runtime: nodejs4.3
+      Runtime: nodejs8.10
```
Comment (SonalOjha) [inline template.yaml]: Make changes to accept this as a param

File: template.yaml [hunk 2]
```diff
@@ -172,7 +172,7 @@ Resources:
     Properties:
       FunctionName: SungardAS-Alerts-Slack
       Handler: index_slack.handler
-      Runtime: nodejs4.3
+      Runtime: nodejs8.10
```
Comment (SonalOjha) [inline template.yaml]: Make changes to accept this as a param

File: template.yaml [hunk 3]
```diff
@@ -194,7 +194,7 @@ Resources:
     Properties:
       FunctionName: SungardAS-Alerts-Teams
       Handler: index_teams.handler
-      Runtime: nodejs4.3
+      Runtime: nodejs8.10
```
Comment (SonalOjha) [inline template.yaml]: Make changes to accept this as a param

File: template.yaml [hunk 4]
```diff
@@ -215,7 +215,7 @@ Resources:
     Properties:
       FunctionName: SungardAS-Alerts-Topic
       Handler: index_sns.handler
-      Runtime: nodejs4.3
+      Runtime: nodejs8.10
```
Comment (SonalOjha) [inline template.yaml]: Make changes to accept this as a param

File: template.yaml [hunk 5]
```diff
@@ -358,7 +358,7 @@ Resources:
         - - "SungardAS-Alerts-Permission"
           - !Ref CloudWatchLogDestinationName
       Handler: index_account_permission.handler
-      Runtime: nodejs4.3
+      Runtime: nodejs8.10
```
Comment (SonalOjha) [inline template.yaml]: Make changes to accept this as a param

File: template.yaml (hunk 1 above)
Comment (bhargavapratik) [reply template.yaml]: Runtime value cannot be a Parameter. still if we need to try this , then we can do while we move code to BitBucket.

File: template.yaml (hunk 2 above)
Comment (bhargavapratik) [reply template.yaml]: Runtime value cannot be a Parameter. still if we need to try this , then we can do while we move code to BitBucket.

File: template.yaml (hunk 3 above)
Comment (bhargavapratik) [reply template.yaml]: Runtime value cannot be a Parameter. still if we need to try this , then we can do while we move code to BitBucket.

File: template.yaml (hunk 4 above)
Comment (bhargavapratik) [reply template.yaml]: Runtime value cannot be a Parameter. still if we need to try this , then we can do while we move code to BitBucket.

File: template.yaml (hunk 5 above)
Comment (bhargavapratik) [reply template.yaml]: Runtime value cannot be a Parameter. still if we need to try this , then we can do while we move code to BitBucket.

**Pull request merged**